model-optimizer/extensions/front/onnx/extract_image_patches_ext.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.ExtractImagePatches import ExtractImagePatches
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.extractor import FrontExtractorOp
+from mo.front.onnx.extractors.utils import onnx_attr
+
+
+class ExtractImagePatchesExtractor(FrontExtractorOp):
+    op = 'ExtractImagePatches'
```

Comment:
Leave a comment here that the ONNX operation is not in the official opset but exist in some models and has input/output in NHWC layout.

Reply:
Done.